cli/cli/helpers/grafloki/docker_grafloki.go

```diff
@@ -18,23 +21,24 @@ import (
 )
 
 const (
-	LokiContainerName    = "kurtosis-loki"
-	GrafanaContainerName = "kurtosis-grafana"
-	lokiReadinessPath    = "/ready"
+	LokiContainerNamePrefix    = "kurtosis-loki"
```

Comment:
Oh, is it expected to have a suffix? If so, I'd expect the prefix to end with something like `-`

Reply:
ah good point